ui_framework/components/global_styles/variables/_animations.scss

```diff
@@ -1,9 +1,10 @@
 // Animations
 
-$globalAnimSlightBounce: cubic-bezier(0.34,1.61,0.7,1);
-$globalAnimSlightResistance: cubic-bezier(0.694, 0.0482, 0.335, 1);
-$globalAnimSpeedVeryFast: 90ms;
-$globalAnimSpeedFast: 150ms;
-$globalAnimSpeedNormal: 250ms;
-$globalAnimSpeedSlow: 350ms;
-$globalAnimSpeedVerySlow: 500ms;
+$kuiAnimSlightBounce: cubic-bezier(0.34,1.61,0.7,1);
+$kuiAnimSlightResistance: cubic-bezier(0.694, 0.0482, 0.335, 1);
+
+$kuiAnimSpeedVeryFast: 90ms;
```

Comment:
Maybe we should use "extra" instead of "very", since we're already using that in our t-shirt sizing?